pkg/controllers/dynakube/extension/tls/conditions.go

```diff
@@ -0,0 +1,3 @@
+package tls
+
+const extensionsTLSSecretConditionType string = "ExtensionsTLSSecret"
```

Comment:
nitpick:
```suggestion
const conditionType string = "ExtensionsTLSSecret"
```

you are in the `tls` package, within the extensions folder, you don't really need to put the whole thing in the name :D

Reply:
[Done](https://github.com/Dynatrace/dynatrace-operator/pull/3994/commits/b6ab3afa45839ba01fce54125c2c97c50025f602)